thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -540,7 +549,7 @@ public Response evaluateAlertFilterByAutoTuneId(@PathParam("autotuneId") long id
   public Response anomalyFunctionReplay(@PathParam("id") @NotNull long functionId,
       @QueryParam("start") @NotNull String replayStartTimeIso, @QueryParam("end") @NotNull String replayEndTimeIso,
       @QueryParam("speedup") @DefaultValue("true") boolean speedup,
-      @QueryParam("tune") @DefaultValue("{\"pValueThreshold\":[0.1,0.05,0.01,0.05,0.01]}") String tuningJSON,
```

Comment:
Why these particular combination of pValueThreshold? try to follow a log-scale for 1, 2, 3 sigma? or should we restrict the default to few cases: like 0.05, 0.01 (2,3 sigma)